### PR TITLE
added HomePage

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
- Added homepage in jsx 
- Created header and footer components
- Added images on home page

image

### CODE REVIEW

**Review by krate27 — APPROVED** [review]
Things look good, though you can use a good code formatter. Also I believe you have not set up API calls so I can see hard coded stuffs. 

Nevertheless UI is good to merge.

**Comment by muskan-gupta-0803** [comment]
> 

yes, it doesn't use api calls as of now. 
Thanks for reviewing.

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* footer styles */
+.footer{
+    position: relative;
+    left: 0px;
+    bottom: 0px;
```
Comment (GeetaPogul) [inline src/components/Footer/Footer.css:5]: avoid using px

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <header className="header">
+        <div className="lilac-navbar">
+            {/* logo */}
+            <a className="logo" href="/index.html">
+                <div className="lilac-nav-logo nav-logo">
+                    Style Store
+                </div>
+            </a>    
+
+            {/*  nav search bar  */}
```
Comment (GeetaPogul) [inline src/components/Navbar/Navbar.js:14]: remove commented code

File: src/pages/Homepage/Homepage.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import FormalShirts from "../../assets/category-img/formal-shirts.jpg";
+import CasualShirts from "../../assets/category-img/casual-shirts.jpg";
+import Jeans from "../../assets/category-img/jeans.jpg";
+import Trousers from "../../assets/category-img/trousers.jpg";
+
+export default function Homepage(){
+    return(
+
+        <main className="main-container">
+          {/*  FIRST SECTION  */}
```
Comment (GeetaPogul) [inline src/pages/Homepage/Homepage.js:12]: here as well

**Review by GeetaPogul — COMMENTED** [review]
good to be merge

**Pull request merged**